### DISCUSSION TITLE
How to Remove Audio Sources?

### DISCUSSION THREAD

**Discussion by ruckusvol** [BimmerGestalt/AAIdrive#605] [General]
I have a few apps that show up in the audio sources list that I would never use. I thought I found a way to disable them in the past but I can't figure it out now. Is there a way to do that?

**Comment by bogdan-calapod** [comment]
You can slide them to the right in the app list and it will disable them

**Reply by ruckusvol** [reply]
Awesome!  That did it.  Thanks so much.
